src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -26,6 +26,17 @@ const deriveLinkPropsFromFormURL = url => {
   return linkProps;
 };
 
+export const deriveLatestIssue = (d1, d2) => {
```

Comment:
Considered moving this function in to a helpers file (since it is exported for testing). However, there is no other component within /find-forms/ that uses the moment library and this function, so I've kept it here. Love to hear if that's how everyone else would have handled it, if not, how would you'd handle it?

Reply:
I typically favor only abstracting once the need arises, I think keeping it here works for now! 💯